uReplicator-Controller/src/main/java/com/uber/stream/kafka/mirrormaker/controller/rest/resources/HealthCheckRestletResource.java

```diff
@@ -10,10 +12,21 @@
  */
 public class HealthCheckRestletResource extends ServerResource {
 
+  private final HelixMirrorMakerManager _helixMirrorMakerManager;
```

Comment:
why start with underscore in the name?

Reply:
Other resource in uReplicator use underscore, I'm aligning with their